README.md

```diff
@@ -102,6 +102,12 @@ end
 To use a different default backend, set `default_backend` to another value (see
 possibilities [below](#backends)).
 
+If you choose `key_value` as `default_backend`, and **MySQL / MariaDB** is your
```

Comment:
The `default_backend` config has changed on the master branch (for v1.0). Could you reword this to:

> If you use `key_value` as your backend, ...

This way the meaning is more general, and I can also cherry-pick this into the 0-8-stable branch.

Reply:
@shioyama 
Thanks for your suggestion. I've pushed a new commit for this.